helpers/addresses.py

```diff
@@ -807,6 +807,12 @@
     },
     "registry": "0xFda7eB6f8b7a9e9fCFd348042ae675d1d652454f",
     "registryV2": "0xdc602965F3e5f1e7BAf2446d5564b407d5113A06",
+    "rewardsLogger": "0xaB2F5E2709286aCe1863DF8ee55475482781F68C",
+    "governanceTimelock": "0x5ddE67e4f0abD7a94E99C037A04A1946f9DbaA36",
```

Comment:
I think we only need one instance of the Timelock here and it would be better to use the same key we use on ETH, `governance_timelock` , for compatibility